docs/tutorials/tqdm_progress_bar.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>Hmmm is this what the expected output looks like? I thought it should show similar to:</p><p><a href="https://i.stack.imgur.com/gpSTt.png" target="_blank">https://i.stack.imgur.com/gpSTt.png</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Resolved, see the newest commit for the change!